test/external.jl

```diff
@@ -50,5 +54,6 @@ close(source_file)
 
 rm(fn1)
 # rm(fn2)
+@debug "external tests did not delete" fn2
```

Comment:
We cannot delete `fn2` or run those above tests because some HDF5 handle is still open. Because of the external link, HDF5 is set to use "weak" closing, so closing the file is not sufficient. Ultimately `h5close` at the end of runtests.jl does close the files and allows for the tempfile to be deleted.

Reply:
I removed the debug message